### ISSUE TITLE
Tidy Requirements

### ISSUE THREAD

**Issue by John-P** [TIA-Lab/tiatoolbox#9]
We are currently using a conda file for setting up the development environment but also have a requirements_dev file around which is used for Travis. Do we even need this anymore if moving to conda for travis? (see #6)

We need to tidy up our requirements files and make sure that the pinned versions are compatible etc. I know that we recently had issues with the conda resolve failing. We need to decide if we want to now pin version numbers for reproducibility when setting up the environment. This would also require making sure that is work on linux, maxOS, and window with the pinned versions.

In summary, issues to resolve:

- [x] Decide if we need a separate development conda file or do we include dev dependencies by default? Note: `setup.py` should define pip dependencies.
- [x] Decide if we should pin version numbers for reproducible resolve of environment or leave blank / use > to allow more flexible resolve. 
- [x] Confirm it works on Linux (Ubuntu >=18.04), macOS, and Windows.
- [x] Decide if we still need `requirements_dev.txt`. If not, make sure CI services e.g. Travis and Read the Docs are set to use conda. See #6.

**Comment by shaneahmed** [comment]
- We don't need separate development conda file.  pip should be fine. 
- We only need to pin version numbers as in [requirements.txt](https://github.com/TIA-Lab/tiatoolbox/blob/docs-readthedocs-config/requirements.txt).
- I have tested these work on Windows (requirements.txt)
- requirements_dev.txt should be fine unless conda is absolutely necessary.
